Sources/Logging/Logging.swift

```diff
@@ -39,20 +39,65 @@ import WASILibc
 /// logger.info("Hello World!")
 /// ```
 public struct Logger {
+    /// Storage class to hold the label and log handler
     @usableFromInline
-    var handler: LogHandler
+    internal final class Storage: @unchecked /* and not actually */ Sendable /* but safe if only used with CoW */ {
+        @usableFromInline
+        var label: String
 
-    /// An identifier of the creator of this `Logger`.
-    public let label: String
+        @usableFromInline
+        var handler: LogHandler
+
+        @inlinable
+        init(label: String, handler: LogHandler) {
+            self.label = label
+            self.handler = handler
+        }
+
+        @inlinable
+        func copy() -> Storage {
+            return Storage(label: self.label, handler: self.handler)
+        }
+    }
+
+    @usableFromInline
+    internal var _storage: Storage
+    public var label: String {
+        get {
+            return self._storage.label
+        }
+        set {
+            if !isKnownUniquelyReferenced(&self._storage) {
+                self._storage = self._storage.copy()
+            }
+            self._storage.label = newValue // mutation is fine now because we just copied
+        }
```

Comment:
This just had a getter and was never settable
```suggestion
        return self._storage.label
```